app/src/main/res/layout/share_close.xml

```diff
@@ -21,6 +21,7 @@
         android:background="@android:color/transparent"
         android:contentDescription="@string/content_description_close_button"
         android:padding="12dp"
+        app:tint="@color/fx_mobile_icon_color_primary"
         app:iconTint="@color/neutral_text"
```

Comment:
I think we can just change `app:iconTint` here, and shouldn't need to keep both `app:iconTint` and `app:Tint`

Reply:
From what I've tested, the `app:iconTint` does not apply on `AppCompatImageView`, but `app:tint` does. I did not find any conclusive documentation about this, but in all the other Fenix layouts `app:tint` is [used](https://github.com/mozilla-mobile/fenix/blob/a1ac0190242ae416472442e3838318d210a6e419/app/src/main/res/layout/tab_preview.xml#L47) for `AppCompatImageView`. So maybe we should drop `app:iconTint` instead and keep `app:tint`?

Reply:
I am okay with dropping `app:iconTint`. I will test this out on Android 6 prior to landing since that seems to be a problematic version when I last worked with icons.

Reply:
While we are addressing this, let's also use `?primaryText` here for now.If you examine our existing icons, you will see text and icons are both linked to `?primaryText`. It would probably be easier to split them out to `@color/fx_mobile_icon_color_primary` in a separate issue.